cmake/modules/FindOrFetchCatch2.cmake

```diff
@@ -13,6 +13,8 @@ if (NOT TARGET Catch2::Catch2)
 
 	if (Catch2_SOURCE_DIR)
 		list(APPEND CMAKE_MODULE_PATH "${Catch2_SOURCE_DIR}/extras")
+  	elseif(Catch2_DIR)
```

Comment:
the commit mentions using vcpkg-provided Catch2 but this block of code is in the from-source clause. Probably needs to be moved outside?